values.yaml

```diff
@@ -53,3 +53,49 @@ reconcile:
   overwriteZk: true
   successfulJobsHistoryLimit: null
   failedJobsHistoryLimit: null
+
+service:
+  type: ClusterIP
+  port: 9000
+
+ingress:
+  ## Set to true to enable ingress record generation
+  ##
+  enabled: true
+
+  ## When the ingress is enabled, a host pointing to this will be created
+  ##
+  host: cmak.local
```

Comment:
Afaik `.local` is reserved zone and its a bad practice to use it.
Just leave `host` empty by default.